save-cloud-charts/save-cloud/values.yaml

```diff
@@ -78,8 +78,6 @@ mysql:
 # Values for dependencies
 grafana:
   enabled: false
-  image:
-    tag: 9.0.1
```

Comment:
why do we remove tag for grafana, but left for rest services (prometheus, promtail)?

Reply:
Yeah, I'll remove it for the rest too. Initially I thought that chart updates may not always include the latest version, but turns out they are updated really often. There is a Renovate's `helm-values` source, but it seems to not be working out of the box.

Reply:
Actually turns out that Renovate expects both keys `image.repository` and `image.tag` to be present to perform update check using `helm-values`. I've added these fields and as you can see in #882 Renovate has triggered on these values.